oh_staff_ui/views_utils.py

```diff
@@ -466,3 +470,50 @@ def run_process_file_command(
     # testing confirms this, though it does close after 30 seconds or so.
     # To be safe, explicitly close this one when done.
     connection.close()
+
+
+def get_listrecords_oai(verb: str, ark: str = None) -> str:
+
+    pi_set = ProjectItem.objects.filter(status__status__iexact="completed")
+
+    if ark:
+        ark = ark.replace("-", "/")
+        pi_set = pi_set.filter(ark=ark)
+
+    verb_element = etree.Element(verb)
+
+    for pi in pi_set:
+        verb_element.append(OralHistoryMods(pi).add_oai_envelope())
```

Comment:
Does the `OralHistoryMods` class need to provide records both with and without an OAI envelope?  If not, or if the norm is with envelope, I think the class should just provide that output automatically, without this external routine telling the class to add the envelope.